source/module_elecstate/test/elecstate_fp_energy_test.cpp

```diff
@@ -55,6 +55,7 @@ TEST_F(fenergy, print_all)
     f_en.print_all();
     std::string output = testing::internal::GetCapturedStdout();
     EXPECT_THAT(output, testing::HasSubstr("total="));
+    EXPECT_THAT(output, testing::HasSubstr("entropy(-TS)="));
```

Comment:
It is confusing to use entropy at one place, but entro at another place. Could you please give explanations about their difference in the code?